docs/README.md

```diff
@@ -0,0 +1,60 @@
+<!--
+# Copyright 2018-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# **Triton Inference Server Documentation**
+
+| [Installation](README.md#installation) | [Getting Started](README.md#getting-started) | [User Guide](README.md#user-guide) | [Additional Resources](README.md#resources) |
+| -------------------------------------- | -------------------------------------------- | ---------------------------------- | ------------------------------------------- |
+
+## **Installation**
+
+See our [Installation Guide](install.md) for details on how to install Model Analyzer.
+
+## **Getting Started**
+
+- Our [Quick Start Guide](quick_start.md) will show you how to use Model Analyzer to profile, analyze and report on a simple PyTorch model.
+- Watch the [Optimizing Triton Deployments](https://www.youtube.com/watch?v=UU9Rh00yZMY) video for a step-by-step guide of how the optimal configuration for a BERT model can be found.
```

Comment:
Follow up with David and Mahan. They may have wanted this embedded (not sure if possible with markdown)

Reply:
I don't think it's possible to embed the video, but you could embed the thumbnail image that links to the video. I tried this, but it looks worse, in my opinion, but maybe there are settings to get the image to format/locate better in the page.

Reply:
You don't need to embed the video in the README. It will not render properly. For Triton's docs, I have it in the new [index.md](https://github.com/triton-inference-server/server/blob/main/docs/index.md) file that used by Sphinx to generate the docs. However, it does not render if you view the file in GitHub. So we can add a link to the video here. If you can add a thumbnail, even better.

Reply:
We can add it proper when we prepare these docs for integration into Triton docs with Sphinx.